bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -179,11 +176,18 @@ impl RestApiConfigBuilder {
 /// REST API configuration.
 #[derive(Clone)]
 pub struct RestApiConfig {
-    pub(crate) binding_socket_addr: SocketAddr,
-    pub(crate) public_routes: Box<[String]>,
-    pub(crate) allowed_ips: Box<[IpAddr]>,
+    /// REST API binding address.
+    pub(crate) bind_socket_addr: SocketAddr,
+    /// JWT salt for REST API.
+    pub(crate) jwt_salt: String,
+    /// Routes that are available for public use and don't need JWT authentication.
+    pub(crate) public_routes: RegexSet,
+    /// Routes that are protected and need JWT authentication.
+    pub(crate) protected_routes: RegexSet,
+    /// Enables/disables the proof-of-work feature on the node.
     pub(crate) feature_proof_of_work: bool,
-    pub(crate) white_flag_solidification_timeout: u64,
+    /// Describes the white flag solidification timeout.
+    pub(crate) white_flag_solidification_timeout: Duration,
```

Comment:
I think those don't need to be `pub(crate)` because of the builder and the already `pub` getters.

Reply:
fixed